src/Tabs/Tabs.js

```diff
@@ -1,111 +1,129 @@
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
-import { BrowserRouter, Link } from 'react-router-dom';
 import React, { Component } from 'react';
 
-export const Tabs = props => {
-    const { children, className, ...rest } = props;
+export const Tab = props => {
+    const { children, className, content, disabled, id, onClick, selectedId, tabContentProps, tabLinkProps, url, ...rest } = props;
```

Comment:
Looks like some of these are missing in the `propTypes` definition. This could be an argument to use classes rather than SFC's.